FirebaseRemoteConfig/Tests/Unit/RCNPersonalizationTest.m

```diff
@@ -60,7 +76,51 @@ - (void)setUp {
         [self->_fakeLogs addObject:bundle];
       });
 
-  [RCNPersonalization setAnalytics:_analyticsMock];
+  _personalization = [[RCNPersonalization alloc] initWithAnalytics:_analyticsMock];
```

Comment:
Could you please add brief comments on what's going on here? Breaking it down to methods with self-documenting names also looks good to me.

Reply:
Done.